### ISSUE TITLE
Dual channel mode

### ISSUE THREAD

**Issue by skyw43** [ololx/sbc-bitpool-expander#7]
Hi, 
Thank you for this very valuable tool. It seems that bit rate doesn't change if I set (or not) dual channel. With bitpool set at 53, this gives me 357kb/s in both cases if I refer to Bluetooth Explorer app in xcode tools. Or maybe the Bluetooth Explorer shows only the value per channel? 

Thanks in advance for your feedback

**Comment by ololx** [comment]
Hi,
thank you for your interest and your feedback. 
The Bluetooth Explorer shows your bitrate correctly. At present, this corresponds to the Joint Stereo channel mode. If there was the Dual Channel mode, then the bitrate would be about 672.0 kbps.
This is my mistake, thanks for finding it. This option is currently inactive. I wanted to add this functionality later, but still have no idea how to do it.
I will remove this checkbox so as not to confuse anyone. I will return it when I implement this functionality.
Thank you. Please close your issue if I answered your question.

**Comment by skyw43** [comment]
Hi thanks for the feedback. I did already boost the bitpool to max allowed by my headphones (53) to try to gain a bit in sound quality. Last step (big step) is to gain to dual stereo. I think the setting "Appel Channel Type" = "DUAL_CHANNEL" you use to set it via check box seems the right one, I tried using command line prompt and it gets accepted by the system. However that doesn't trigger the dual stereo at all as bitrate keeps at 357kbps unfortunately. I have no idea but it sounds like a bug at Apple side. Any view on this from Apple support on your side ? Hope we could find how to set it right as this is really a huge gain ;-)
